modules/search-service-impl/src/main/java/org/opencastproject/search/endpoint/SearchRestService.java

```diff
@@ -465,35 +457,33 @@ public Response getEpisode(
 
     boolean invalidSeries = false;
     if (seriesName != null) {
-      if (seriesService == null) {
-        return Response.status(Response.Status.NOT_IMPLEMENTED).build();
-      }
-      DublinCoreCatalogList result;
+      SearchResult result = new SearchResultImpl();
       try {
-        result = seriesService.getSeries(new SeriesQuery().setSeriesTitle(seriesName));
-      } catch (SeriesException e) {
+        SearchQuery seriesSearch = new SearchQuery();
+        seriesSearch.includeSeries(true)
+                    .includeEpisodes(false)
+                    .withText(seriesName)
```

Comment:
Won't this search for the series field in all text fields? I think you want to look for the title only?

Reply:
It's roughly the same way the search works when you search /search/series.json with a series title as query text. It does search all fields this way, but by setting includeseries = true and includeepisodes = false this does seem to work as expected. I tested this by also creating an event with the same title as the series title, and this does not conflict. Tbh I also did it this way because I didn't find any functions to directly search on the series title field.

Reply:
It wouldn't conflict with episodes, but it could conflict with other series. 
If you have something line:

```yaml
- title: biology 101
  description: we do things
- title: abstract art
  description: the 101 of painting things
```

And you search using `sname=101`, you might get `abstract art` as an answer, wouldn't you?
My question is why you search in all text and not only in the title field?

Or am I misinterpreting things and `withText()` is actually only searching in the title field?

Reply:
No, valid point, it was indeed search all fields. Fixed this now, and tested it as suggested with the series description to make sure this doesn't conflict anymore.